pod/recorder/views.py

```diff
@@ -70,3 +89,121 @@ def add_recording(request):
 
     return render(request, "recorder/add_recording.html",
                   {"form": form})
+
+
+def reformat_url_if_use_cas(request, link_url):
```

Comment:
Can you explain ?

Reply:
It's a code of Loic, according to the comment : "Pointing to the URL of the CAS allows to reach the already authenticated form URL "

Reply:
ok, we can keep it